source/agora/consensus/validation/Block.d

```diff
@@ -435,7 +435,7 @@ unittest
     auto gen_hash = GenesisBlock.header.hashFull();
 
     GenesisBlock.txs.each!(tx => utxos.put(tx));
-    auto block = GenesisBlock.makeNewBlock(makeChainedTransactions(gen_key, null, 1));
+    auto block = GenesisBlock.makeNewBlock(genesisSpendable().map!(txb => txb.sign()).array());
```

Comment:
```suggestion
    auto block = GenesisBlock.makeNewBlock(genesisSpendable().map!(txb => txb.sign()));
```

`makeNewBlock` accepts an input range, so there's no need to call `.array` - it's an unnecessary allocation.